templates/CRM/common/formButtons.tpl

```diff
@@ -28,6 +28,23 @@
    by button and name. crmBtnType grabs type keyword from button name (e.g. 'upload', 'next', 'back', 'cancel') so
    types of buttons can be styled differently via css. *}
```

Comment:
This comment block seems like it is for the `$form.buttons` array written below your changes. can we move this to the correct place? i.e, after `{if linkButtons}` block?

Reply:
@jitendrapurohit Yes you are right. I've moved the comment to the right place